### ISSUE TITLE
Release a 0.7.3 version of dask-jobqueue

### ISSUE THREAD

**Issue by guillaumeeb** [dask/dask-jobqueue#505]
I know there have only been a few changes since 0.7.2, but I'd like to issue a new release for two main reasons:

- https://github.com/dask/dask-jobqueue/pull/503, our Dask users have their logs getting polluted a lot since june Dask release, this seems to be really uncomfortable for developing and debugging things.
- I think it will make this project more alive to have some regular release, last one was seven months ago.

Any opinion on this @andersy005 @lesteve @jacobtomlinson @jrbourbeau?

And do we want to wait for some ongoing work (maybe #502? I don't see any other small changes), or try to fix some small issues first (not found anything with a quick glance)?

**Comment by lesteve** [comment]
I am fine with doing a 0.7.3 soon (maybe without waiting for #502). Thanks for pushing this forward!

**Comment by lesteve** [comment]
By the way, ideas to make this project more alive (as you said) would be more than welcome! Maybe organising small development sprints from time to time?

Personally I have been busy with other things recently so I did not have time to do much related to Dask ...

**Comment by andersy005** [comment]
> By the way, ideas to make this project more alive (as you said) would be more than welcome! Maybe organising small development sprints from time to time?

👍🏽 So looking forward to this... Would this be at a recurring time e.g. once a month or once a quarter depending on people's availability?

**Comment by guillaumeeb** [comment]
> By the way, ideas to make this project more alive (as you said) would be more than welcome! Maybe organising small development sprints from time to time?

We're all busy with too many things... But I could try to book half a day once a quarter!

**Comment by wshanks** [comment]
Just a release with #503 and #506 would be really great for me. The warnings are frequent enough to cause performance issues with my workloads.

**Comment by lesteve** [comment]
I have uploaded released 0.7.3 on PyPI: https://pypi.org/project/dask-jobqueue/0.7.3/. Any problems let me know!

I think it should be picked up automatically at one point in conda-forge.

**Comment by lesteve** [comment]
0.7.3 is available in conda-forge:
```
❯ conda search -c conda-forge dask-jobqueue=0.7.3
Loading channels: done
# Name                       Version           Build  Channel             
dask-jobqueue                  0.7.3    pyhd8ed1ab_0  conda-forge 
```
